templates/favorites.html

```diff
@@ -24,6 +24,14 @@ <h1>Your Favorites:</h1>
     {% endif %}
     {% endwith %}
 
+
+
+    <form action="/add_recommendations" method="POST">
```

Comment:
I'll create a flask WTform similar to BookInfoFormAdd (but with the addition of a receiver_email field and recommendation submit button) to prevent csrf_token errors and to keep things uniform.